src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,21 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @ManyToMany
```

Comment:
- when use toMany association you can get lazyInitializationException, because its' default fetch type is LAZY, let' use EAGER for now to avoid this